provider/resource_rediscloud_acl_rule_test.go

```diff
@@ -12,32 +12,9 @@ import (
 	"testing"
 )
 
-func TestAccRedisCloudAclRule_DataSourceForDefaultRule(t *testing.T) {
-	// This rule already exists
-	const testName = "Read-Write"
```

Comment:
was this test being duplicated or something?

Reply:
This one was being placed in its own location file.  Some of the tests needed to be renamed and this caused a lint naming error.